MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -141,7 +140,7 @@ public class CarPlayNavigationViewController: UIViewController {
         styleManager!.styles = self.styles
         
         makeGestureRecognizersResetFrameRate()
-        resumeNotifications()
+        resumeNotifications(by: navigationService)
```

Comment:
minor suggestion: how about `observeNotifications(by:)` since that's closer to what we're doing, semantically?